lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -85,7 +85,7 @@ def setup_rabbitmq_connection_config
         config :tls_certificate_path, :validate => :path, :obsolete => "This setting is obsolete. Use ssl_certificate_path instead"
 
         # TLS certificate password
-        config :tls_certificate_password, :validate => :string, :obsolete => "This setting is obsolete. Use ssl_certificate_password instead"
+        config :tls_certificate_password, :validate => :password, :obsolete => "This setting is obsolete. Use ssl_certificate_password instead"
```

Comment:
Can we even remove `tls_certificate_path`  and `tls_certificate_password`? They are not used in the source at all.

Reply:
Yes, lets remove them since they are obsolete now.